addons/web/static/src/webclient/commands/command_palette.js

```diff
@@ -1,15 +1,23 @@
 /** @odoo-module **/
 
-import { registry } from "@web/core/registry";
+import { registry } from "../../core/registry";
+import { KeepLast } from "../../core/utils/concurrency";
+import { debounce } from "../../core/utils/timing";
 import { useAutofocus } from "../../core/autofocus_hook";
-import { isMacOS } from "../../core/browser/feature_detection";
 import { useHotkey } from "../../core/hotkey_hook";
 import { scrollTo } from "../../core/utils/scrolling";
+import { fuzzyLookup } from "../../core/utils/search";
+import { useService } from "../../core/service_hook";
```

Comment:
Here's my 2 cents:
Import style shouldn´t be "@web/core/..." ?
Didn´t we choose to use relative paths only when changing 1st level directory ?